challenges/identification/I3_benchmark/I3_specification.md

```diff
@@ -0,0 +1,113 @@
+# I3 benchmark specification
+
+## Synopsis
+
+Benchmark to compare output of tool's RNAseq based site identification to an established atlas of polyA-sites (PolyASite, PolyA_DB). The goal is to calculate precision with respect to data from existing databases for several different distance thresholds.
+
+Input data:
+
+1. Poly(A) sites identified based on RNAseq data using the benchmarked tool
+2. Poly(A) sites identified in PolyASite database
+3. Poly(A) sites identified in PolyA_DB database
+
+Based on the input data the following metrics are computed:
+
+1. Precision = (TP/(TP+FP))
+
+TP - true positives - PAS identified by the tool and present within X nucleotides of PAS identified in the database  
+FP - false positives  - PAS identified by the tool and not present within X nucleotides of PAS identified in the database
+
+The metrics should be calculated for the following thresholds of distance between PAS identified by the tool and PAS present in databases:
+
+- 50 nt
+- 100 nt
+- 200 nt
+
+
+## General info
+
+* **Challenge:** Identification
+* **Identifier:** I3
+
+## Inputs
+
+| # | Format | Link | Example data |
+  | --- | --- | --- | --- |
+  | 1 | BED | [Specification][spec-bed] | [Link][in1] |
+  | 2 | BED | [Specification][spec-bed-polyAsite] | [Link][in2] |
```

Comment:
Would it be useful to have these pre-conformed? That is, should we have adapters that conform polyASite and PolyA_DB to standard BED6 format? It doesn't seem like we'll use any of the other data.

Reply:
If we want to add pre-conformed example inputs, then we'd also have to modify actual database files before for the workflow to run, right? I think it would be better to just leave it for the evaluation teams to process the database files inside the workflow. **Or** we could add a separate issue for database file processing and then this benchmark would be database-agnostic.

Reply:
I was thinking the latter, but we could get to it when it comes up.